#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#

float32 loadTime

float32 resolution

uint32 widthCells
uint32 heightCells

float32 widthM
float32 heightM

bool negate

srslib_framework/Pose origin

string occupancyFilename

float32 thresholdFree
float32 thresholdOccupied
